source/sections/footer.liquid

```diff
@@ -7,9 +7,11 @@
     {% form 'customer', class: 'form' %}
       {{ form.errors | default_errors }}
```

Comment:
If a success message just appears on the page, without aria-live, then a screen reader user would have no idea that this happened, especially since the success message appears in a location in the DOM that the user has already gone past, to submit the form.  adding an aria-live here will announce the message to these users when it is revealed.